examples/chef/devices/rootnode_colortemperaturelight_hbUnzYVeyn.matter

```diff
@@ -179,7 +179,7 @@ server cluster OnOff = 6 {
     kAcceptOnlyWhenOn = 0x1;
   }
 
-  readonly attribute boolean onOff = 0;
+  readonly nosubscribe attribute boolean onOff = 0;
```

Comment:
nosubscribe status is unexpected and is seen nowhere else except rootnode_pump in chef, which I believe it is a bug as well. Please update zap file to not have this.